app/models/audio_file_template.rb

```diff
@@ -0,0 +1,22 @@
+# encoding: utf-8
+
+class AudioFileTemplate < BaseModel
+  belongs_to :audio_version_template
+  acts_as_list scope: :audio_version_template
+
+  before_validation :set_defaults, on: :create
+
+  validates :length_minimum,
+            presence: true,
+            numericality: { less_than_or_equal_to: :length_maximum }
+
+  validates :length_maximum,
+            presence: true,
+            numericality: { greater_than_or_equal_to: :length_minimum }
+
+  def set_defaults
+    self.label ||= 'segment'
+    self.length_minimum ||= 0
+    self.length_maximum ||= 0
```

Comment:
min or max of 0 are ignored, treated as there being nothing set, so you can't have a min of 0.
this is consistent with v3 implementation.